aptos-move/framework/supra-framework/sources/multisig_voting.spec.move

```diff
@@ -0,0 +1,494 @@
+spec supra_framework::multisig_voting {
+    /// <high-level-req>
+    /// No.: 1
+    /// Requirement: The proposal ID in a voting forum is unique and always increases monotonically with each new proposal
```

Comment:
Did you just copy the `spec` file from original voting? If so, delete it. This won't work with the prover.

Reply:
Done